talentlink-connector/src/com/axonivy/connector/talentlink/features/TalentlinkQueryStringFilter.java

```diff
@@ -0,0 +1,27 @@
+package com.axonivy.connector.talentlink.features;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.regex.Matcher;
+
+import javax.ws.rs.client.ClientRequestContext;
+import javax.ws.rs.client.ClientRequestFilter;
+import javax.ws.rs.core.UriBuilder;
+
+import ch.ivyteam.ivy.environment.Ivy;
+
+public class TalentlinkQueryStringFilter implements ClientRequestFilter{
+	
+	private final static String api_key = Ivy.var().get("talentlink-connector.apikey");
```

Comment:
The api_key resolved from Ivy.var() should not live on a `static final` field. Or users especially on the engine will not have the chance to change the configured key without re-starting the engine. Just read it and pass it directly to the URI builder without holding the value anywhere.